src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -0,0 +1,44 @@
+package com.game.Battleship.models;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+
+    private int size;
+    private List<Cell> boardField;
+
+    public Board(){
+        this.size = 10;
+        this.boardField = generateBoardField();
+    }
+
+    private List<Cell> generateBoardField(){
```

Comment:
I'm not 100% sure on this so I want you to look this up, but I would expect the private methods to be at the end of the file.

Reply:
Done